package-res/reportviewer/report.html

```diff
@@ -116,7 +116,10 @@
 
 <div id="reportArea" class="pentaho-transparent">
   <div id="reportPageOutline">
-    <iframe id="reportContent" frameborder="0"></iframe>
+    <form id="hiddenReportContentForm" method="post" target="reportContentIframe">
```

Comment:
Absolutely! The reason is because of the iframe. There are things happening once the report is loaded etc. that trigger the functionality between PUC and that iframe. This approach leaves all of that alone, POST's the data as required and loads it in the existing iframe. Everyone listening for that or interacting with it behaves as before.